my-laravel-app/public/assets/js/order-details.js

```
@@ -12,18 +12,31 @@ document.addEventListener("DOMContentLoaded",function(e){var t,a,r=document.quer
         {data:"qty"},
         {data:"id"}],
         
-        columnDefs:[{className:"control",searchable:!1,orderable:!1,responsivePriority:2,targets:0,render:function(e,t,a,r){return""}},{targets:1,orderable:!1,searchable:!1,responsivePriority:3,checkboxes:!0,checkboxes:{selectAllRender:'<input type="checkbox" class="form-check-input">'},render:function(){return'<input type="checkbox" class="dt-checkboxes form-check-input">'}},{targets:2,responsivePriority:1,searchable:!1,orderable:!1,render:function(e,t,a,r){var s=a.product_name,n=a.product_info,a=a.image;let o;return`
-        <div class="d-flex justify-content-start align-items-center text-nowrap">
-          <div class="avatar-wrapper">
-            <div class="avatar avatar-sm me-3">${o=a?`
-          <img src="${assetsPath}img/products}" lass="rounded-2">
-        `:`<span class="avatar-initial rounded-2 bg-label-${["success","danger","warning","info","dark","primary","secondary"][Math.floor(6*Math.random())]}">${(s.match(/\b\w/g)||[]).slice(0,2).join("").toUpperCase()}</span>`}</div>
+        columnDefs:[{className:"control",searchable:!1,orderable:!1,responsivePriority:2,targets:0,render:function(e,t,a,r){return""}},{targets:1,orderable:!1,searchable:!1,responsivePriority:3,checkboxes:!0,checkboxes:{selectAllRender:'<input type="checkbox" class="form-check-input">'},render:function(){return'<input type="checkbox" class="dt-checkboxes form-check-input">'}},{targets:2, responsivePriority:1, searchable:!1, orderable:!1, 
```

Biome (1.9.4)

[error] This property value named checkboxes is later overwritten by an object member with the same name.

Overwritten with this value.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property value named checkboxes


(lint/suspicious/noDuplicateObjectKeys)